internal/installcfg/installcfg.go

```diff
@@ -136,23 +138,23 @@ func getBasicInstallConfig(cluster *common.Cluster) *InstallerConfigBaremetal {
 		PullSecret: cluster.PullSecret,
 		SSHKey:     cluster.SSHPublicKey,
 		ImageContentSources: []struct {
-			Mirrors []string `yaml:"mirrors"`
-			Source  string   `yaml:"source"`
+			Mirrors []string `yaml:"mirrors", envconfig:"MIRROR_REPO"`
+			Source  string   `yaml:"source", envconfig:"MIRROR_SOURCE_IMAGE_1", envconfig:"MIRROR_SOURCE_IMAGE_2"`
 		}{
 			{
 				Mirrors: []string{
-					"local.registry:5000/ocp4",
+					os.Getenv("MIRROR_REPO"),
```

Comment:
Not a good idea to read from env randomly in the middle of the code. How are rest of the variables read from the file?

Reply:
Rest ones are read as struct.field but I am not sure how will it work when I change `Source: os.Getenv("IMAGE_CONTENT_SOURCE_2"),` to `Source: InstallerConfigBaremetal.Source,` I will test it on a working machine.

Reply:
@sacharya The dependency `"github.com/kelseyhightower/envconfig"` currently has no support for a slice of structs but they do have an open PR here https://github.com/kelseyhightower/envconfig/pull/170. Till then should we continue to read from env variables? `AdditionalTrustBundle` can be changed not to read from env var but `ImageContentSources` will still need to be. WDYT?

Reply:
With reference to the above comment, I am proceeding with reading from env var.